tlsfuzzer/messages.py

```diff
@@ -764,6 +764,46 @@ def new_calculate_mac(mac, seqnumBytes, contentType, data,
 
     return generator
 
+def fuzz_encrypted_message(generator, substitutions=None, xors=None):
+    """Change arbitrary bytes of the AEAD sealed block"""
+    def new_generate(state, self=generator,
+                     old_generate=generator.generate,
+                     substitutions=substitutions,
+                     xors=xors):
+        """Monkey patch to modify ciphertext block"""
+        msg = old_generate(state)
+
+        old_send = state.msg_sock._recordSocket.send
+
+        self.old_send = old_send
+
+        def new_send(message, padding, old_send=old_send, substitutions=substitutions,
+                     xors=xors):
+            """Monkey patch for the send method of msg socket"""
+
+            """
+                message.data is the encrypted tls record, e.g.
+                message.data = aead_encrypt(plain); defined by aead suite
+                message.data = encrypt(plain + padding) + mac; etm
+                message.data = encrypt(plain + mac + padding); mte
+            """
```

Comment:
this will not be interpreted as a doc string, needs to be together, with the first line ("Monkey patch...") on its own, empty line and the "message..." following. It should not be indented